runtime/doc/builtin.txt

```diff
@@ -4261,6 +4261,19 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2}				*getregion()*
+		The result is a list of strings where each item is a line
+		from {pos1} to {pos2}.
```

Comment:
The docs doesn't mention that `pos1` and `pos2` are swapped if `pos1` is after `pos2`.

Reply:
OK.  I have added it.